jest.config.ts

```diff
@@ -25,7 +26,9 @@ const reactMarkdownEsModules = [
   "zwitch",
 ];
 
-const esModules = [...reactMarkdownEsModules].join("|");
+const esModules = [...reactMarkdownEsModules, "@buoysoftware/anchor-.+"].join(
```

Comment:
In order to get the modal spec to pass, we had to add anchor-* to the transformIgnorePatterns. We did this at the root level, but it stands to reason we could update this to be for individual packages or components instead. WDUT?